src/Services/ProcedureService.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * ProcedureService
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Yash Bothra <yashrajbothra786gmail.com>
+ * @copyright Copyright (c) 2020 Yash Bothra <yashrajbothra786gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Services;
+
+use OpenEMR\Common\Uuid\UuidRegistry;
+use OpenEMR\Validators\BaseValidator;
+use OpenEMR\Validators\ProcessingResult;
+
+class ProcedureService extends BaseService
+{
+
+    private const PROCEDURE_TABLE = "procedure_order";
+    private const PATIENT_TABLE = "patient_data";
+    private $uuidRegistery;
+
+    /**
+     * Default constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct(self::PROCEDURE_TABLE);
+        $this->uuidRegistery = new UuidRegistry([
+            'table_name' => self::PROCEDURE_TABLE,
+            'table_id' => 'procedure_order_id'
+        ]);
```

Comment:
Three cheers for the new UuidRegistry feature :)

![2DV](https://user-images.githubusercontent.com/278968/88761778-ab3ab280-d124-11ea-8fda-57404fb21c35.gif)

Reply:
Yay :)

![minions](https://user-images.githubusercontent.com/47683556/88791318-29479980-d1b7-11ea-918a-b452c29919c2.gif)

Reply:
lets make it three :)

![image](https://user-images.githubusercontent.com/278968/88866007-416ee700-d1be-11ea-9dfb-7ab1cfe7c8df.png)